### PR TITLE
SEO Page

### PR DESCRIPTION
**Pull request by ekonstantinidis**
Some basic progress (colors etc for demonstration only):
screen shot 2015-07-19 at 12 32 44

### CODE REVIEW

**Comment by JakeSidSmith** [comment]
New logo & updated styles.

screen shot 2015-07-20 at 18 05 51
screen shot 2015-07-20 at 18 05 59

**Comment by JakeSidSmith** [comment]
Favicon! :smiley: 
screen shot 2015-07-20 at 18 27 57

**Comment by ekonstantinidis** [comment]
Niice! :cookie:

**Comment by JakeSidSmith** [comment]
Here's some more screenshots.

screen shot 2015-07-20 at 19 48 43
screen shot 2015-07-20 at 20 09 30
screen shot 2015-07-20 at 20 12 20

**Comment by ekonstantinidis** [comment]
Looks awesomeee :ship: :ship: :ship:

File: index.html [hunk 1]
```diff
@@ -19,7 +25,137 @@
...
+      </div>
+
+    </section>
+
+    <section class="section">
+
+      <div class="container-fluid footer">
+        <div class="container">
+          <div class="row">
+            <div class="col-md-6">
```
Comment (JakeSidSmith) [inline index.html]: These could probably be col-sm-6 and still fit. Or this one slightly larger on small screens.

**Comment by JakeSidSmith** [comment]
:rocket:

**Pull request merged**